teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super(47, 'teacher')
```

Comment:
- [ ] Same here kindly consider adding all the parameters to the constructor, also make sure to add parameters to the parent constructor instead of using specific values. the code should be something like this 👇🏼 
```ruby
def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
```